<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export</class>
 <widget class="QDialog" name="Export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="garminChk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Garmin Connect</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stravaChk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Strava</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="FITChk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FIT File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="targetDir">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fitButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shareButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Share</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="allButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Export all new activities</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="todayButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Only sync activities from today. If used with the &amp;quot;flag all&amp;quot; option this will prevent a huge backlog of workouts uploading.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Export today's activities</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="flagBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you have a lot of old activities you can auto-flag them as already transferred.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Flag all as transferred</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
